kmodes/kprototypes.py

```diff
@@ -162,12 +168,13 @@ def fit(self, X, y=None, categorical=None):
             self.n_init,
             self.verbose,
             random_state,
-            self.n_jobs
+            self.n_jobs,
+            sample_weights,
         )
 
         return self
 
-    def predict(self, X, categorical=None):
+    def predict(self, X, categorical=None, sample_weights=None):
```

Comment:
Sample weights are typically only used during fitting of the model, so this looks a bit weird to me. I would suggest removing it from the `predict` method.

Reply:
Great point!
[20e8dd6](https://github.com/nicodv/kmodes/pull/171/commits/20e8dd6657adb445a8486da17f647eff9ef8903a)